torchmetrics/retrieval/mean_average_precision.py

```diff
@@ -61,13 +61,13 @@ class RetrievalMAP(RetrievalMetric):
         >>> from torchmetrics import RetrievalMAP
         >>> indexes = torch.tensor([0, 0, 0, 1, 1, 1, 1])
         >>> preds = torch.tensor([0.2, 0.3, 0.5, 0.1, 0.3, 0.5, 0.2])
-        >>> target = torch.tensor([False, False, True, False, True, False, False])
+        >>> target = torch.tensor([False, False, True, False, True, False, True])
 
         >>> map = RetrievalMAP()
         >>> map(indexes, preds, target)
-        tensor(0.7500)
+        tensor(0.7917)
         >>> map.compute()
-        tensor(0.7500)
+        tensor(0.7917)
```

Comment:
just remove the `map.compute()` part as it is the same

Reply:
Yes